src/components/Photo/DocumentLiveCapture.js

```diff
@@ -82,7 +83,7 @@ export default class DocumentLiveCapture extends Component<Props, State> {
       renderError,
       documentType
     } = this.props
-    const { hasBecomeInactive, hasCameraError } = this.state
+    const { hasBecomeInactive, hasCameraError, isLoading } = this.state
```

Comment:
`isLoading` seems to be inconsistent with `isCapturing` on the selfie capture.

Reply:
document also uses `screenshot` which is the (potential) async function that selfie uses (which is the reason for `isLoading`

Reply:
reading the ticket, this work on document and selfie capture is out of scope. one recommend way is to just remove this form the diff

Reply:
rereading, `isLoading` is never changed, this seems like dead code that just needs to be cleanedup.

Reply:
final reread 😅 , `isLoading` and `isCapturing` are the same, so they should be named the same.

Reply:
Renamed `isLoading` to `isCapturing` since it's not much work to do anyway